html/field-studies.html

```diff
@@ -129,7 +129,8 @@
                 ],
                 render_result_record: function(options, record)
                 {
-                    return template(record).trim();
```

Comment:
would it make sense to empty out all that code as well? If we need it back we can always get it from an earlier revision. So remove lines 24 to 66